packages/twenty-server/src/core/user/user.entity.ts

```diff
@@ -72,6 +73,10 @@ export class User {
   })
   defaultWorkspace: Workspace;
 
+  @Field()
+  @Column()
+  defaultWorkspaceId: string;
```

Comment:
Is there a particular reason why we haven't included the `defaultWorkspaceId` on `user.entity.ts`?